### DISCUSSION TITLE
Best way to organize tables

### DISCUSSION THREAD

**Discussion by mweber06** [brianpetro/obsidian-smart-connections#317] [Q&A] [answered]
The vault I have is *very* table oriented with some tables having nested tables, some having uneven columns and rows, and most tables referencing/linking to multiple notes with tables in them as well. It's sort of a mess of tables everywhere that not even markdown can fully support. What's the best way to organize this data structure for smart connections to operate at peak performance?

I've been rewriting everything into heading hierarchies, but not even I can make sense of things anymore so I'm not sure what SC can do. Does the embedding process even parse tables, especially ones with links, and ones written in html or with the advance tables addon? Thanks.

**Comment by RealRavens** [comment, marked as answer]
Well without seeing it, I'm not sure. Technically you have freedom with how you design your notes, even if you do it with a table referencing a bunch of links. You could, if not done already, make all the notes in each cell a transcluded note and give it a generic ID. 

But anyway, for "peak" performance as long as your tables don't grow to big under a particular topic/idea/train of thought, it should work relatively fine. What do I mean? The AI can better answer questions or respond to prompts with your notes if your notes are easy to work with puzzle pieces. Or I guess in a more direct and sophisticated way. That's why breaking them down into headers or multiple notes holding some sort of atomic idea or thought is recommended. However, the AI can still pick apart larger notes mind you (try to keep the notes related and not too all over the place), it just works better with more pieces. Now will there be more parsing options? Well that's tricky to know, I guess that's up to @brianpetro, but I understand being hesitant to add new ones, cause it's sorta based off how Obsidian already divides up notes, so maybe in the future with Obsidian updates you can get table cell note separation.

Good luck, happy note taking.
